src/merge_split.cpp

```diff
@@ -89,9 +89,9 @@ Rcpp::List ms_plans(int N, List l, const uvec init, const uvec &counties, const
         // NOTE: different signs than above b/c of how Metropolis proposal has
         // transition ratio flipped relative to the target density ratio
         prop_lp -= calc_gibbs_tgt(districts.col(i), n_distr, V, distr_1, distr_2,
-                                  pop, constraints, counties, n_cty);
+                                  pop, target, g, constraints);
```

Comment:
I think we need to though? Flip doesn't draw spanning trees and only takes a county arg through here.

I think it's best to break it now since it hasn't been made a stable feature yet.

Reply:
do any of our 50-states analyses use the soft constraint?  If not then I'm ok with breaking it now I guess

Reply:
Seems that we do not. We made good use of the pick_county_muni set up to use the graph constraint (which is pretty effective for small n districts)

Reply:
ok then